packages/styled/src/index.js

```diff
@@ -5,7 +5,7 @@ import { tags } from './tags'
 // bind it to avoid mutating the original function
 const newStyled = styled.bind()
 
-tags.forEach(tagName => {
+tags.forEach((tagName: string) => {
```

Comment:
shouldnt this be easily inferrable?

Reply:
It does't infer it for some reason, I see it marked as `any`

Reply:
Got it, `tags.js` didn't have the flow comment header